src/Headless/CLIViewer.cpp

```diff
@@ -163,7 +164,9 @@ void CLIViewer::setCamera( Ra::Core::Utils::Index camIdx ) {
         Ra::Engine::RadiumEngine::getInstance()->getSystem( "DefaultCameraManager" ) );
 
     if ( camIdx.isInvalid() || cameraManager->count() <= size_t( camIdx ) ) {
-        m_camera = new Ra::Core::Asset::Camera( m_parameters.m_size[0], m_parameters.m_size[1] );
+        m_ownCamera = std::make_unique<Ra::Core::Asset::Camera>( m_parameters.m_size[0],
```

Comment:
Why not adding the created camera to the  `CameraManager` that manage ownership of cameras ? Even if there are some deficiencies in the `CameraManager` interface and the way it is used (whose fix is out of scope of this PR), this could be done as follow : 
```c++
{
        auto e      = m_engine->getEntityManager()->createEntity( "Default Camera" );
        auto camera = new Scene::CameraComponent( e, "Camera" );
        camera->initialize();
        cameraManager->addCamera( camera );
        cameraManager->activate( cameraManager->getCameraIndex( camera ) );
        m_camera = cameraManager->getActiveCamera();
        m_camera->setFOV( 60.0_ra * Ra::Core::Math::toRad );
        /// the same lines as below  to initialize the camera ...
        m_camera->setZFar( zfar );
    }
```
The method should then return the camera index of the active (the one created here) camera so that client can use it in other call to this method.

Reply:
Maybe, since I'm not sure what this naked new is supposed to do and why it arrives here, my solution is to make it smart, but another solution is surely better. 
Commit/fix welcome.

Reply:
OK, I'll do that ASAP